### PR TITLE
Chunky Turtle Logic Rework 1.19.2

### PR DESCRIPTION
**Pull request by zyxkad**
#576

1.19.2 version

### CODE REVIEW

**Comment by zyxkad** [comment]
Alr, a lot of things need to change for 1.19.2, will work on it later

**Comment by SirEndii** [comment]
Is this PR ready for review?

**Comment by zyxkad** [comment]
yes

File: src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/turtles/TurtleChunkyUpgrade.java [hunk 1]
```diff
@@ -34,10 +35,16 @@ protected ChunkyPeripheral buildPeripheral(@NotNull ITurtleAccess turtle, @NotNu
 
     @Override
     public void update(@NotNull ITurtleAccess turtle, @NotNull TurtleSide side) {
-        //Add a chunk to the Chunk Manager every 10 ticks, if it's not already forced.
-        //The turtle can move, so we need to do that.
+        // Add a chunk to the Chunk Manager every 10 ticks, if it's not already forced.
+        // The turtle can move, so we need to do that.
         super.update(turtle, side);
         if (APConfig.PERIPHERALS_CONFIG.enableChunkyTurtle.get()) {
+            updateTick++;
```
Comment (SirEndii) [inline src/main/java/de/srendi/advancedperipherals/common/addons/computercraft/turtles/TurtleChunkyUpgrade.java]: This currently does not seem to do anything
Shouldn't it be removed or commented out?

**Review by SirEndii — CHANGES_REQUESTED** [review]

**Review by SirEndii — APPROVED** [review]

**Pull request merged**